utils/crabz/wrapper.py

```
@@ -0,0 +1,38 @@
+    fmt = get_format(file_path, ignore_compression=False)
+    print(fmt)
+    if fmt == "bgzip":
+        return "bgzf"
+    return fmt
+
+
+available_compression_formats = {"gzip", "bgzf", "mgzip", "zlib", "deflate", "snap"}
+
+input_file_format = crabz_format(str(snakemake.input))
+if input_file_format in available_compression_formats:
+    extra += f" --decompress"
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)